posts/2019/07/ibis-python-data-analysis-productivity-framework.ipynb

```diff
@@ -0,0 +1,424 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ibis: Python Data Analysis Productivity Framework\n",
+    "\n",
+    "Ibis is a library pretty useful on data analysis tasks. That allows you work locally over a schema, like create filter, add columns, apply math operations etc and when you want to get the result of the expression you created, Ibis compiles that and makes a request to the remote server (remote storage and execution systems like Hadoop components or SQL databases). Its goal is to simplify analytical workflows and make you more productive.\n",
```

Comment:
Should this contain something like "provides a pandas-like API to ..."? That may explain it better than the schema statement to people who haven't heard of Ibis before

Reply:
sounds good .. I am changing that! thanks